design-system/Blockies.ts

```diff
@@ -0,0 +1,132 @@
+// Copyright © 2021 The Radicle Upstream Contributors
+//
+// This file is part of radicle-upstream, distributed under the GPLv3
+// with Radicle Linking Exception. For full terms see the included
+// LICENSE file.
+
+// Copyright (c) 2019, Ethereum Name Service
```

Comment:
Seeing our copyright at the top and then later the original copyright below seems odd to me.
But I'm also not sure what the proper way of dealing with copyright notices is in this case.

Do you know @cloudhead?

Reply:
What about it is odd? The order or the fact of having two copyright notices? It's typical to have a copyright notice for each author who worked on the file.

Reply:
Hadn't noticed multiple notices before, just wanted to double-check. 👌